flict/__main__.py

```diff
@@ -98,7 +86,7 @@ def parse():
     description = description + "DESCRIPTION\n  " + PROGRAM_DESCRIPTION + "\n\n"
 
     epilog = ""
```

Comment:
In a future refactor I would consider using `+=` in the following lines or to just create a multi line block.